packages/react-relay/relay-hooks/useRefetchableFragmentNode.js

```diff
@@ -516,10 +518,21 @@ if (__DEV__) {
     getInitialIDAndType(
       memoRefetchVariables: ?Variables,
       fragmentRefPathInResponse: $ReadOnlyArray<string | number>,
+      fragmentNode: ReaderFragment,
+      componentDisplayName: string,
```

Comment:
nit: pass `identifierField` instead of recomputing it. i would also default to `id` if identifierField is null/undefined